expo/gbe/functions.py

```diff
@@ -145,3 +146,24 @@ def available_volunteers(event_start_time):
         if event_start_time in window_range:
             windows.append(window)
     return conf.Volunteer.objects.filter(available_windows__in=windows)
+
+
+def get_events_and_windows(conference):
```

Comment:
Fair enough.  It's actually easy enough to move it right now if we think a decent guideline is:
- querying tables in other application models is higher on the list of things to avoid than accessing parts of objects from other application models since abstraction is better that way
- better to add more imports from scheduler functions 

The refactor is pretty easy and decidedly easier to do now than later.